### PR TITLE
MIME-DB tests

### PR DESCRIPTION
**Pull request by bcomnes**
~~WIP. These asserts still need work probably.~~ This is ready for review!

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

File: test/src/mime.js [hunk 1]
```diff
@@ -1,8 +1,132 @@
...
+  }
+
+  for (const item of right) {
+    if (left.includes(item)) {
+      // @ts-ignore
+      results.add(item)
+    }
+  }
+
+  return Array.from(results.entries())
```
Comment (chicoxyzzy) [inline test/src/mime.js:25]: is it correct that it's an array of entries and not values?

Comment (jwerle) [reply test/src/mime.js:25]: `Set.prototype.entries()` returns an [`Iterator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator) (_Set Entries_) that is an [`Iterable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols), which we convert to an array with `Array.from()`. This is desired

Comment (chicoxyzzy) [reply test/src/mime.js:25]: The return value is in JSDoc is wrong than, it's `[number, string][]` on this line and `string[]` in JSDoc

Comment (chicoxyzzy) [reply test/src/mime.js:25]: maybe we can simplify it to
```js
return left.filter(value => right.includes(value));
```
?

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
the `intersection` implementation looks incorrect

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**